src/ui/ServerAdministration/index.tsx

```diff
@@ -321,6 +335,29 @@ class ServerAdministrationContainer
     }
   }
 
+  private async showPartialRealmWarning() {
+    const show = localStorage.getItem(DONT_SHOW_PARTIAL_WARNING_KEY) !== 'true';
+    if (show) {
+      return new Promise(resolve => {
+        electron.remote.dialog.showMessageBox(
+          electron.remote.getCurrentWindow(),
+          {
+            message:
+              'You are opening a partial Realm created for a specific device. To ensure other divices do not write to this it will be opened in read-only mode.',
```

Comment:
divices -> devices.

But perhaps the wording could be better around "To ensure other divices do not write to this".
Perhaps "client" is a better term than "device"?
Perhaps just:
"You are opening a partial Realm created for a specific client. You can only view the content in read-only mode."